src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -45,20 +49,15 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
   const { artwork, me, relay } = props
   const { saleArtwork } = artwork
   const { sale } = saleArtwork
-
+  const { environment } = relay
   const { trackEvent } = useTracking()
 
   function createBidderPosition(maxBidAmountCents: number) {
-    return new Promise(async (resolve, reject) => {
-      commitMutation<ConfirmBidCreateBidderPositionMutation>(
-        relay.environment,
-        {
-          onCompleted: data => {
-            resolve(data)
-          },
-          onError: error => {
-            reject(error)
-          },
+    return new Promise<ConfirmBidCreateBidderPositionMutationResponse>(
+      async (resolve, reject) => {
```

Comment:
is this mix of Promise and `async` necessary?

Reply:
Removed, things seem to be working without it.